test/integration/chronokvs/chronokvs_integration_test.cpp

```diff
@@ -0,0 +1,334 @@
+#include "chronokvs.h"
```

Comment:
the current style of included header files are 1. system/std lib header files; 2. ChronoLog header files that are NOT in the same dir; 3. ChronoLog header files within the same dir. Three groups are separated by an empty line.